terraform/playground/terraform-playground.sh

```diff
@@ -58,7 +58,7 @@ generate_azure_private_workspace_name () {
     # becomes 'foobar'.
     # Below command errors out with the azure error message if the azure user
     # is not signed-in.
-    WORKSPACE=$(az ad signed-in-user show | jq -cr .userPrincipalName | cut -d'@' -f1 | sed 's/[^a-zA-Z0-9]//g' | cut -c 1-20)
+   WORKSPACE=$(az ad signed-in-user show | jq -cr .userPrincipalName | cut -d'@' -f1 | sed 's/[^a-zA-Z0-9]//g' | cut -c 1-20 | tr '[:upper:]' '[:lower:]')
```

Comment:
Please fix the indentation (add one space character before `WORKSPACE=...`)

Reply:
Also, for the sake of symmetry, please consider adding the following on line 150:
`exit_unless_command_exists tr`

And the following comment on line 57:
`# - tr converts the string to lower case`

Reply:
Also, not enforced currently, but you should try to follow the commit message guidelines at: https://github.com/tiiuae/ghaf/blob/main/CONTRIBUTING.md#commit-message-guidelines.
In this case, while making the above suggested changes, you could also re-formulate the commit subject-line.